ci/fih_test_docker/fi_tester_gdb.sh

```diff
@@ -40,6 +40,13 @@ function skip_instruction {
 
     cat >commands.gdb <<EOF
 target remote localhost: 1234
+file $IMAGE_DIR/bl2.axf
+b boot_go_for_image_id
```

Comment:
Nit: I think this should be able to be a `b boot_go_for_image_id if image_id != 0`, which makes the logic slightly nicer. Could well be wrong though, I hardly ever use conditional breakpoints these days.